src/languageservice/nonTSqlKeywords.ts

```diff
@@ -0,0 +1,694 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+/**
+ * This list was built by finding keywords in the MySql documentation,
+ * and  PostGreSql documentation that are not present in the TSQL documentation
+ * of keywords.
+ */
+export const keywords: Set<string> = new Set([

```

Comment:
What if these keywords appear in object names? It's possible for keywords like 'cost' to be used as object names. With the current logic, this might trigger the prompt. I'm not sure what the best solution would be, but it's something to consider.

Reply:
Maybe we should trim some likely candidates for object names from this list?